core/utils/mailbox_prom.go

```diff
@@ -0,0 +1,72 @@
+package utils
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var mailboxLoad = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	Name: "mailbox_load_percent",
+	Help: "Percent of mailbox capacity used",
+},
+	[]string{"appID", "name", "capacity"},
+)
+
+const mailboxPromInterval = 10 * time.Second
```

Comment:
Probably could use a more generic `promInterval` and share use of it globally (with jitter). Node status reporting comes to mind: https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/client/pool.go#L161

Reply:
Hmm it seems messy to borrow a chain specific setting. Is there anything else at the root level?

Reply:
Reduced to 5s with jitter for now.